src/routes/get-started/developers/DevelopersSetup.tsx

```diff
@@ -113,7 +113,17 @@ export default function DevelopersSetupPage() {
         code={`@import "@abgov/web-components/index.css";`}
       />
 
-      <GoADivider mt="2xl" mb="xl" />
+        <GoACallout mt={"xl"}>
```

Comment:
I don't think the Callout should be necessary. Just make it a normal step like everything else. Reasoning being, is that anyone going to the setup steps now, will be starting from scratch, so they'll be using the latest version by default (hopefully), so they should automatically be told to do this, not have to make a decision based on their version.